CRM/Event/Form/ManageEvent/Repeat.php

```diff
@@ -24,12 +23,19 @@ class CRM_Event_Form_ManageEvent_Repeat extends CRM_Event_Form_ManageEvent {
    */
   protected $_parentEventEndDate = NULL;
 
+  /**
+   * @return int
+   */
+  protected function getEventId() {
```

Comment:
Add a "getter" function so we don't have to access class properties directly and it is clearer that $this->_id = event ID.